safe_transaction_service/safe_messages/serializers.py

```diff
@@ -134,12 +144,15 @@ def get_signature_type(self, obj: SafeMessageConfirmation) -> str:
 
 
 class SafeMessageResponseSerializer(serializers.Serializer):
+    created = serializers.DateTimeField()
+    modified = serializers.DateTimeField()
     safe = eth_serializers.EthereumAddressField()
     message_hash = eth_serializers.Sha3HashField()
     message = serializers.JSONField()
     proposed_by = eth_serializers.EthereumAddressField()
     description = serializers.CharField()
     confirmations = serializers.SerializerMethodField()
+    prepared_signature = serializers.SerializerMethodField()
```

Comment:
The naming on this one is a bit strange but I also don't have better suggestions... `computedSignature` or just `signature`? Maybe something that we can ask the team.

Reply:
How about `concatenated_signature`?

Reply:
I don't have an strong opinion, but I'm between `computedSignature`, `preparedSignature` or `proposedSignature`. `concatenatedSignature` sounds that we are just pasting it together, and there's further logic involved

Reply:
Let's go with `preparedSignature` I'd say (no strong opinion). Maybe we can ask in the team chat but shouldn't block this PR.

The only requirement I'd say is that we should be aligned on this before a production release.